scripts/install-go.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -xe
+
+destination=$1
+version=$(grep "^go " go.mod |awk '{print $2}')
```

Comment:
It might cause a problem if there are lines in go.mod that begin with "go".

Reply:
It doesn't mater if the file contains other lines with "go", it will always use the first line with "go" which is the right one
The first line in go.mod is always the package declaration
The second line in go.mod is always the go version